### ISSUE TITLE
Dask XGboost impossible to reproduce same results

### ISSUE THREAD

**Issue by sergiocalde94** [dask/dask#4697]
I´m using dask for implementing a data pipeline with dask dataframes and dask ml in a Yarn Cluster.

When I build an XGBoost model, the results are always different, even if I manually fix a seed with da.random.seed().

```
import dask_xgboost as dxgb


params = {'objective': 'binary:logistic', 'n_estimators': 420,
           'max_depth': 5, 'eta': .01,
          'subsample': .8, 'colsample_bytree': .8,
          'learning_rate': .05, 'scale_pos_weight': 1}

bst = dxgb.train(client, params, fitted.transform(X), y)
```

Is it possible to reproduce the results of a dask model like the one in local using sklearn instead of dask ml???

**Comment by sergiocalde94** [comment]
The problem is when I run the model in cluster mode (not local). It´s a Yarn Cluster as I mentioned before.

**Comment by jcrist** [comment]
Closing, as double posted in the dask-xgboost repo: https://github.com/dask/dask-xgboost/issues/37. Will continue discussion there.
